sdks/java/io/google-cloud-platform/src/test/java/org/apache/beam/sdk/io/gcp/spanner/SpannerWriteIT.java

```diff
@@ -116,9 +117,12 @@ public void setUp() throws Exception {
   }
 
   private String generateDatabaseName() {
-    String random = new RandomStringGenerator.Builder().build()
-        .generate(MAX_DB_NAME_LENGTH - 1 - options.getDatabaseIdPrefix().length())
-        .toLowerCase();
+    String random =
+        new RandomStringGenerator.Builder()
+            .filteredBy(CharacterPredicates.LETTERS)
```

Comment:
Hmm, I neglected to push my changes. LETTER is too broad. DIGIT might work. But instead I will use a codepoint range.